spec/system/admin/reports_spec.rb

```diff
@@ -130,9 +131,14 @@
         breakpoint.synchronize { "continue after unlocked" }
       end
 
+<<<<<<< HEAD
       click_on "Go"
 
       expect(page).to have_content "FIRST NAME LAST NAME BILLING ADDRESS EMAIL"
+=======
+      expect(page).to have_selector "#report-table table"
+      expect(page).to have_content "First Name Last Name Billing Address Email"
+>>>>>>> 0a69712a7b (Fixes case on assertions, on failing specs)
```

Comment:
This looks like a mistake.

Reply:
But it seems to have been fixed later.

Reply:
Yeah... Sorry. Indeed, this might have happened a couple of times. This branch lived for quite a long time and was fixed for conflicts/rebased countess times. I'm not sure if it would not have been a better approach to open this branch, at a later stage. Maybe a topic for discussion if we get to do a retro on BUU.